app/models/heads_of_term.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class HeadsOfTerm < ApplicationRecord
+  belongs_to :planning_application
+  has_many :reviews, as: :owner, dependent: :destroy, class_name: "Review"
+  has_many :terms, extend: TermsExtension, dependent: :destroy
+  has_many :validation_requests, through: :terms
+
+  accepts_nested_attributes_for :terms, allow_destroy: true
+  accepts_nested_attributes_for :reviews
+
+  after_update :maybe_create_review
```

Comment:
`create_review_if_review_required`
Or something a bit more descriptive than maybe

Reply:
Perhaps `after_update :create_review, if: :should_create_review`.

Then the `should_create_review` method only needs the boolean part, rather than also calling `create_review` itself.